index.html

```diff
@@ -287,6 +291,24 @@ <h2>About Myself</h2>
           </div>
         </div>
       </section>
+
+      <!--
+        Contact section
+      -->
+      <section class="contact-form">
+        <h2>Contact me</h2>
```

Comment:
- [ ] You have done great so far, :+1:  your form works as expected. But there are few UI changes you need to make before moving to the next milestone.
- [ ] As part of the requirement for this milestone is to replicate the given designs much as you can, I think you need to increase the font size/Boldness of this text, your seems to be smaller than that of the Figma design. :+1: :+1:  
- - Your design:
![Screenshot from 2023-06-13 15-58-40](https://github.com/LUCASFRANKINC/module1_project/assets/107632780/40bc4de2-df8a-483f-8580-e996e924b846)

- - Figma design:
![Screenshot from 2023-06-13 15-58-18](https://github.com/LUCASFRANKINC/module1_project/assets/107632780/4ce0b7e0-e302-443f-8561-90426c86a939)